docs/standard/base-types/custom-numeric-format-strings.md

```diff
@@ -44,23 +46,25 @@ You can create a custom numeric format string, which consists of one or more cus
 |Other|All other characters|The character is copied to the result string unchanged.|68 ("# °") -> 68 °|  
```

Comment:
This table row should reference a new **Character literals** section, as other rows do.
The same is true for the line 44: row about literal string delimiters.